### PR TITLE
#1021 Populate sms_sender_id in the notification object for SMS.

### PR DESCRIPTION
**Pull request by kalbfled**
# Description

These changes add the "sms_sender_id" and associated Flask relationship fields to the Notification and NotificationHistory models.  POST requests to send an SMS notification should require the sms_sender_id field, which can be "null".  If sms_sender_id is null, the notification should be sent with a default value associated with the service.

The bulk of the changes in this PR are modifying the unit tests.  The changes also include a significant amount of comments, annotations, and cleanup that I made while crawling through the code base.

The ticket specifically is about /v2/notifications/sms, but changes to get unit tests to pass for that route broke the v1 route, which is in /app/notifications.  I had to update both routes to get all the tests passing.

This PR also adds 2 new CA certificates to /certs.  We started getting user-flows failures after VA Profile updated a certificate signed with CA certs we didn't have, and this addition addresses that change.

Deployed: https://github.com/department-of-veterans-affairs/notification-api/actions/runs/3851163623

#1021

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- [x] All unit tests pass.  I altered relevant tests to inspect sms_sender_id.
- [x] Manual testing in dev by POSTing to /v2/notifications/sms.  This seems to work when sms_sender_id is null and when it is a valid UUID.

I have not manually tested POSTing with a valid recipient_id because I'm not sure where to find one.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/utils.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
+# TODO - This file contains multiple functions that contain import statements?  Is this to avoid a circular
+# dependency?  If not, import only at the beginning of the file.
```
Comment (cris-oddball) [inline app/utils.py:2]: Please create a ticket for new TODOs and then reference the ticket number in this TODO comment.

Comment (kalbfled) [reply app/utils.py]: done

**Review by cris-oddball — APPROVED** [review]

**Review by k-macmillan — APPROVED** [review]
This is all great. You took care of a lot of the hardcoded strings too which was very nice to see. Good stuff!

**Review by EvanParish — APPROVED** [review]
There were definitly more changes required than I thought there would be. Good job!

**Pull request merged**